force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -175,7 +260,11 @@ public with sharing class VariablesDataTypesOperators {
      * @return The average of the three numbers.
      */
     public static Double averageOfThreeNumbers(Integer a, Integer b, Integer c) {
-        return null; // Replace null with the variable you used to store the result
+        Integer total numbers = 3;
+        Double sum = (double)(a + b + c);
+
+
+         return sum / total; // Replace null with the variable you used to store the result
     }
```

Comment:
In this case you know that 3 is a constant value because your method takes 3 arguments.  Can you  get ride of the variable total and do all this on one line?